app/models/concerns/best_practices/scopes.rb

```diff
@@ -4,4 +4,16 @@ module BestPractices::Scopes
   included do
     scope :ordered, -> { order obsolete: :asc, name: :asc }
   end
+
+  module ClassMethods
+    def visible
+      setting = Current.organization.settings.find_by name: 'hide_obsolete_best_practices'
+
+      if setting.value != '0'
+        where(obsolete: false)
+      else
+        all
+      end
```

Comment:
Esto es una maña, pero prefiero que siempre la primera sea "positiva", así que:
```suggestion
      if setting.value == '0'
        all
      else
        where(obsolete: false)
      end
```